### ISSUE TITLE
Add pages missing from Sitemap

### ISSUE THREAD

**Issue by unclesam2024** [web-infrastructure/doc_architecture#38]
Where a [Structured Sitemap](https://en.wikipedia.org/wiki/Sitemaps) is defined, it should contain all public pages from this website to help search engines discover them quickly.

This action looks for pages which were tested but not found in the current Sitemap for the website. This could be because the Sitemap is wrong or out of date. On rare occasions, it could be because:

1. Private pages were tested (e.g. behind a login) which should not be in the Sitemap
2. The URL scheme for pages tested was different from what is defined in the Sitemap
